src/summary.py

```
@@ -223,161 +228,410 @@ async def unreadsummary(
+    print(error)
+    await ctx.followup.send(ERROR.format(error))
+
+
+async def discord_command_summarize_all(ctx, 
+                                        time_period: str = Option(str, "Time period",            required=False, default="1d"), 
+                                        tag:         str = Option(str, "Tag to filter channels", required=False, default=None),
+                                        category:    str = Option(str, "Category to filter channels", required=False, default=None)
+                                        ):
+    now = datetime.now()
+    time_to_look_back = time_for_dating_back(now, time_period)
+    time_to_look_back_for_context = now - timedelta(days=CONTEXT_LOOKBACK_DAYS)
```

Ruff (0.8.2)

Local variable `time_to_look_back_for_context` is assigned to but never used

Remove assignment to unused variable `time_to_look_back_for_context`

(F841)